pyatv/mrp/protocol.py

```diff
@@ -133,11 +149,9 @@ def stop(self):
         self._initial_message_sent = False
         self._outstanding = {}
         self.connection.close()
+        self._state = ProtocolState.NotConnected
```

Comment:
This is the logical state here, but I think I would like to introduce another state (e.g. `Stopped`) as a way to prevent re-use of the protocol instance. Because now you can call `start` again after `stop` and I don't see any point of doing that, so it's better to block in order to find potential bugs.